deployer/cli.py

```diff
@@ -16,6 +17,13 @@
 )
 
 
+def _convert_string_to_list(full_str: str) -> list:
+    """
+    Take a SPACE-DELIMITED string and split it into a list
+    """
+    return full_str.split(" ")
```

Comment:
Sorry for not making it clear!

Yes i was thinking to either directly reference str.split instead of the new custom function which then is deleted, or to clarify it makes sense to have a dedicated function that does something slightly different (narrowing the split function to only accepts spaces)

Reply:
I think `str.split` is a function on the class `str` that has the same function signature I think.

```
In [1]: str.split("ab cd")
Out[1]: ['ab', 'cd']
```

The comment about `str.split` was just a non-important detail - please proceed in any way!